.gitignore

```diff
@@ -93,3 +92,7 @@ test.tap
 # Xcode workspaces and project folders
 *.xcodeproj
 *.xcworkspace
+
+# Visual Studio Code
+.vscode/
+jsconfig.json
```

Comment:
I found the PR: https://github.com/nodejs/node/pull/6963

... I think is just better to add this kind of common files to `.gitignore` that asking people to add this to the global ignore. **Just because this way we don't need to mention again something about this topic and really this don't hurt anyone**